build.gradle

```diff
@@ -39,6 +39,7 @@ allprojects {
 		if (version.endsWith('BUILD-SNAPSHOT') || project.hasProperty('platformVersion')) {
 			maven { url 'https://repo.spring.io/libs-snapshot' }
 		}
+		maven { url 'https://repository.apache.org/content/groups/staging' }
```

Comment:
Since `0.11.0` is already [GA](https://github.com/apache/kafka/releases/tag/0.11.0.0),  we don't need this repo any more.